practice_app/apps/user/views.py

```diff
@@ -0,0 +1,95 @@
+from django.shortcuts import render
+from django.http import JsonResponse
+import json
+
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from .serializers import UserSerializer
+from .models import User
+
+@api_view(['GET'])
+def apiOverview(request):
+    api_urls = {
+        'List':'/user-list/',
+        'Detail':'/user-detail/',
+        'Post':'/user-create/',
+    }
+    return Response(api_urls)
+
+@api_view(['GET'])
+def userList(request):
+    users = User.objects.all()
+    serializer = UserSerializer(users, many=True)
+    return Response(serializer.data)
+
+@api_view(['GET'])
+def userDetail(request, pk):
```

Comment:
You can add if-else statements for invalid usernames.

Reply:
I have already implemeted this after first commit. Thank you again for your attention.